katello/katello-change-hostname

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/ruby
+
+require "socket"
+require "optparse"
+require "rubygems"
+require 'yaml'
+
+raise 'Must run as root' unless Process.uid == 0
+
+DEFAULT_SCENARIO = File.basename(File.readlink("/etc/foreman-installer/scenarios.d/last_scenario.yaml")).split(".")[0]
+DEFAULT_SCENARIO == "katello" ? DEFAULT_PROGRAM = "foreman" : DEFAULT_PROGRAM = DEFAULT_SCENARIO
```

Comment:
Nitpick but this can be confusing to read vs what I would think is the more standard form of the ternary operator assignment:

```
DEFAULT_PROGRAM = DEFAULT_SCENARIO == "katello" ? "foreman" : DEFAULT_SCENARIO
```